fordarnold/Papel
# #171309759 Setup Coveralls for test coverage reporting

#### What does this PR do?
Implement test coverage reporting into our project using Coveralls, a web service to help you track your code coverage over time, and ensure that all your new code is fully covered by automated tests.

#### Description of Task(s) to be completed?
- Enable the project GitHub repo in Coveralls dashboard.
- Setup a Coveralls config file in project root.
- Add Coveralls badge to README file.
- Setup npm scripts for running test coverage.

#### How should this be manually tested?
Check the README of the project repository on GitHub. You will see a CI badge displaying the current "coverage" status.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#171309759

#### Screenshots (if appropriate)

#### Questions:
